sdk/error_test.go

```diff
@@ -150,23 +142,15 @@ func TestMultiError(t *testing.T) {
 	mError.Append(errFourth)
 
 	assert.Equal(t, "TestMultiError: internal server error (caused by: my first error), TestMultiError: wrong request (from: my second error), TestMultiError: internal server error (caused by: hidden info: my third error), TestMultiError: already exists (from: my fourth error)", mError.Error())
-	httpErr := ExtractHTTPError(mError, "fr")
```

Comment:
Can you check in english please instead of fr cause here we want to make sure that the message is well printed for user no matter the language ?

Reply:
fixed